NEWS.md

```diff
@@ -1,4 +1,14 @@
-# fitzRoy (development version)
+# fitzRoy 1.5.1
```

Comment:
As above, leave it as fitzroy (development version) to show that changes aren't yet on CRAN. We can increment it when I submit it to CRAN